test/csuite/timestamp_abort/main.c

```diff
@@ -56,9 +56,10 @@ static char home[1024];			/* Program working dir */
  * Each worker thread creates its own records file that records the data it
  * inserted and it records the timestamp that was used for that insertion.
  */
+#define	INT_SESSION_CNT 36
 #define	INVALID_KEY	UINT64_MAX
 #define	MAX_CKPT_INVL	5	/* Maximum interval between checkpoints */
-#define	MAX_TH		12
+#define	MAX_TH		200
```

Comment:
Why do we need 36 extra sessions? Where did that number come from? The code inside WiredTiger already adds in an extra 20 for internal WT threads. The program only uses 3 internal threads to do other work.
Since it is a constant, I'd prefer it to be defined here as a constant rather than down in the code. Something like:
```
#define SESSION_CNT MAX_TH + 3 /* Maximum threads plus program worker threads */
```